swagger/src/main/scala/org/http4s/rho/swagger/models.scala

```diff
@@ -196,18 +196,53 @@ object models {
     , delete           : Option[Operation] = None
     , patch            : Option[Operation] = None
     , options          : Option[Operation] = None
+    , head             : Option[Operation] = None
     , parameters       : List[Parameter]   = Nil
     , vendorExtensions : Map[String, Any]  = Map.empty
     ) {
-
+    
+    import com.fasterxml.jackson.annotation.{ JsonPropertyOrder, JsonIgnore }
+    
+    @JsonPropertyOrder(Array("get", "post", "put", "delete", "options", "patch", "head"))
+    class ComplementaryPath extends jm.Path {
```

Comment:
Ok, I think that makes sense to me. I would have to do it after the hours.
If you think this is a trivial enough change, feel free to just make it on
top of mine.

On Wednesday, August 26, 2015, Bryce Anderson <notifications@github.com>
wrote:

> In swagger/src/main/scala/org/http4s/rho/swagger/models.scala
> <https://github.com/http4s/rho/pull/103#discussion_r37975580>:
>
> >      , parameters       : List[Parameter]   = Nil
> >      , vendorExtensions : Map[String, Any]  = Map.empty
> >      ) {
> > -
> > +
> > +    import com.fasterxml.jackson.annotation.{ JsonPropertyOrder, JsonIgnore }
> > +
> > +    @JsonPropertyOrder(Array("get", "post", "put", "delete", "options", "patch", "head"))
> > +    class ComplementaryPath extends jm.Path {
>
> I think we just need to bump our swagger-core dep to 1.5.3 and we get
> these head operations, right? (See here
> <https://github.com/swagger-api/swagger-core/blob/v1.5.3/modules/swagger-models/src/main/java/io/swagger/models/Path.java#L54>
> )
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/http4s/rho/pull/103/files#r37975580>.
>